tests/core/test_simulation.py

```diff
@@ -779,3 +781,359 @@ def test_export(speos: Speos):
         sim_second.export(export_path=str(Path(test_path) / "export_test"))
 
     remove_file(str(Path(test_path) / "export_test"))
+
+
+@pytest.mark.skipif(IS_DOCKER, reason="COM API is only available locally")
```

Comment:
@StefanThoene as COM API is not covered in docker, unittests are skipped. 
Could you please review the test cases if any situation is missed:
currently tests: 
3d sensor, photometric, radiometric, colorimetric,
irradiance sensor, photometric, radiometric, colorimetric, spectral.